frontend/src/components/EditableSelect.module.scss

```
@@ -9,6 +9,15 @@
   margin-top: 0;
 }
 
+.editButtonDiv {
+  text-align: end;
+  width: min-content;
+}
+
+.value {
+  margin: 0;
+  flex-grow: 1;
+}
 .selectWrapper {
```

```suggestion
}

.selectWrapper {
```

nitpick: I'd add a line between toplevel classes